### ISSUE TITLE
[Bug] Uncategorized unsubmitted segments aren't being skipped.

### ISSUE THREAD

**Issue by DShip91** [ajayyy/SponsorBlock#962]
Version 3.0.2 of the extension broke unsubmitted segments and they now no longer get skipped. As of version 3.2, this bug has not been fixed.

**Comment by DShip91** [comment]
Any idea what's causing this bug? Because it's still not fixed as of version 3.6.1.

**Comment by ajayyy** [comment]
It has been fixed for me, no one else has reported this for a while

**Comment by DShip91** [comment]
You're right. I had to do some digging but I was able to find out what was different and apply it to all of my other unsubmitted segments.

**Comment by NatoBoram** [comment]
I have a segment that's uncategorized because I don't want to submit it by accident, but it's no longer being skipped :/
